### ISSUE TITLE
igraph - map size of nodes to LoC

### ISSUE THREAD

**Issue by nfultz** [aaronrudkin/werner#11]
It would be slick if long functions were larger and short functions were smaller. LoC is one measure, out degree would be another.

**Comment by aaronrudkin** [comment]
Yeah, absolutely. I basically did a bare minimum mapping from my data to qgraph. I had originally tried igraph, which I know is more popular, but couldn't easily make vertices repel each other, so there were issues in dense parts of the graph -- I also didn't like its default label handling compared to qgraph's. But I'm agnostic going forward in terms of qraph versus igraph versus statnet or whatever.

The other hurdle here is of course that right now the main functions just return, respectively, a dumb list of lists, and a dumb matrix. We'd obviously want to augment those data with more useful measures besides a function call map.

**Comment by nfultz** [comment]
Alternatively, see tidygraph and ggraph.
